AccountingSystem/ConfigTemplate.cfg

```diff
@@ -25,4 +25,14 @@ Services
       }
     }
   }
-}
\ No newline at end of file
+}
+Agents
+{
+  NetworkAgent
+  {
+    MaxCycles = 0
```

Comment:
The NetworkAgent is meant to work continuously and indefinitely long. The default value for MaxCycles is 500 what, taking in to account the pooling time equal to 60s, means just 8 hours of work.